cmd/kube-proxy/app/server_others.go

```diff
@@ -271,9 +271,9 @@ func newProxyServer(
 		}
 		proxymetrics.RegisterMetrics()
 	} else if proxyMode == proxyModeIPVS {
-		klog.V(0).Info("Using ipvs Proxier.")
+		klog.V(0).InfoS("Using ipvs Proxier.")
```

Comment:
you've removed all trailing dots but this :smile:

Reply:
In pursuit of perfection, I will fix it. Thanks very much.